bool isConnected
bool running
bool isInRemoteControl

string robotmode
string programState
string loadedProgram
string popup

string safetymode
string safetystatus

# string getSerialNumber
# string getRobotModel
# string polyscopeVersion
